### ISSUE TITLE
Auto clicks/ repeated clicks missing

### ISSUE THREAD

**Issue by Wakeboardtiger** [nicymike/serflings#35]
https://github.com/nicymike/serflings/issues/20#issuecomment-667454787

**Comment by Wakeboardtiger** [comment]
still no autoclick?

**Comment by nicymike** [comment]
Yes, still working on a solution. My plan was to first switch to Java 17 (Serflings version 2), because this will change some internal stuff.

**Comment by Wakeboardtiger** [comment]
Any solution yet?

**Comment by nicymike** [comment]
Sorry, no solution yet. I am currently working on a different issue.
